.github/workflows/benchmarks.yml

```
@@ -0,0 +1,47 @@
+name: "Benchmark workflow"
+on:
+  # Push and update benchmarks when a commit is merged into master.
+  push:
+    branches:
+      - master
+  # Trigger benchmark test on this PR's commit against master.
+  pull_request:
+    branches:
+      - master
+permissions:
+  # Push benchmark performance graph to gh-pages branch.
+  contents: write
+  deployments: write
+jobs:
+  benchmark:
+    name: Performance regression check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+      - run: go version
+      - name: Build go-algorand
+        run: scripts/travis/build.sh
+      - name: Run benchmark
```

@algochoi Leaving a suggestion motivated by the following question:  When the alert threshold is breached, how will folks triage the issue?

In an attempt to keep things _simple_ to start, I'm thinking a couple of inline sentences can help provide context.  The _suggestion_ attempts to explain the benchmark's significance, which hopefully provides a clue for triaging.

Feel welcomed to modify the wording as you see fit.  I'd prefer to add _some_ version of the text.  If you'd rather omit it entirely, I'm open to hearing your thoughts.

```suggestion
      # `BenchmarkUintMath` - Serves as a proxy for AVM `eval` performance.  Performance degradations suggest either or both:  (1) op code degradation, (2) `eval` degradation.  (2) suggests a broader performance issue. 
      - name: Run benchmark
```